{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -2,8 +2,10 @@ Django==2.1.2
 psycopg2-binary==2.7.5
 dj-database-url==0.5.0
 whitenoise==4.1
+channels==2.1.3
+channels_redis==2.3.0
+service_identity==17.0.0
```

Comment:
I'm not exactly clear on what this package does, or whether it belongs in the template.